py/plugins/gitleaks.py

```diff
@@ -34,8 +34,7 @@
 
 GITLEAKS_LOG = "/builddir/gitleaks-capture.log"
 
-FILTER_CMD = "csgrep '%s' --mode=json --warning-rate-limit=%i > '%s'"
-
+FILTER_CMD = "csgrep '%s' --mode=json --warning-rate-limit=%i --gitleaks-limit-msg-len=%i > '%s'"
```

Comment:
The `csdiff` option introduced by https://github.com/csutils/csdiff/pull/134 is called `--limit-msg-len` so this code won't work.  I haven't tested that locally but the following patch should fix it:
```suggestion
FILTER_CMD = "csgrep '%s' --mode=json --warning-rate-limit=%i --limit-msg-len=%i > '%s'"
```